save-backend/Backend-API.md

```diff
@@ -282,10 +282,10 @@ The response format will look like:
 ```
 
 
-If you would like to rerun some of your executions, you can use `/rerunExecution` endpoint:
+If you would like to rerun some of your executions, you can use `/run/re-trigger` endpoint:
 
 ```bash
-curl -X POST "${SAVE_CLOUD_URL}/api/v1/rerunExecution?id=${executionId}" \
```

Comment:
We should also refactor `save-api` someday 😵, now it should compiles at best and not working at all

Reply:
agree, raised: https://github.com/saveourtool/save-cloud/issues/1061